### PR TITLE
dev -> master PR 

### PR DESCRIPTION
**Pull request by BarryDigby**
## PR checklist

- [x] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
  - [ ] If you've added a new tool - add to the software_versions process and a regex to `scrape_software_versions.py`
  - [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/circrna/tree/master/.github/CONTRIBUTING.md)
  - [ ] If necessary, also make a PR on the nf-core/circrna _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [x] Make sure your code lints (`nf-core lint .`).
- [x] Ensure the test suite passes (`nextflow run . -profile test,docker`).
- [x] Usage Documentation in `docs/usage.md` is updated.
- [x] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [x] `README.md` is updated (including new tool citations and authors/contributors).

***

#26

### CODE REVIEW

**Comment by apeltzer** [comment]
Hi @BarryDigby  - I think this needs a couple of more things to be able to merge what is on `dev` to the master branch:

- Latest `nf-core` template (2.5.1 is current now, that should also fix some of the issues with linting)
- One round of reviews from @nf-core/core (2 required for making a release on master), probably with some more fixes required but looking at what has been done already not likely to be that much anymore

**Comment by BarryDigby** [comment]
Hi @apeltzer,

I recently deleted the nf-core template branches when cleaning up my repo, can I generate the latest template branch locally using `nf-core`? 

Best, 

Barry

**Comment by apeltzer** [comment]
Hi @BarryDigby  - yes you can: https://nf-co.re/developers/sync#manual-synchronisation

Might be a bit more cumbersome than if the TEMPLATE would still be available, but still feasible I think. The merge into dev then takes a bit more work but should still fix all of the issues I think - @nf-core/core anything I've overlooked?

**Comment by ewels** [comment]
If the pipeline was created using `nf-core create` in the first place then there should be no need. Just go back to the first commit, where all code is vanilla template without customisation, and make the new branch from there.

```bash
git checkout master
git log --reverse
git checkout <first commit hash>
git branch TEMPLATE
git push
```

**Comment by ewels** [comment]
Note that if you can resurrect the deleted branch, that's better. Every time you've done a template sync you've solved merge conflicts and doing them cumulatively should be easier than coming from the first template version. However, the approach above ☝🏻 should still be mich easier than a fully manual sync.

If you have local copies you can push them, if you deleted locally you can try `git reflog`, if you have old PRs you can sometimes resurrect stuff. Git is quite good at keeping things around..

**Comment by ewels** [comment]
Ah, looks like you have a `TEMPLATE` branch on your fork, so don't even need to do the git commands above, can just push that branch to the nf-core repo: https://github.com/BarryDigby/circrna/tree/TEMPLATE

There's also [this fork](https://github.com/ARCCA/circrna/commits/TEMPLATE) which has a bunch of the automated sync commits in. That's what we want ideally, as if those same commits (with commit hashes) are in `dev` or `master` then that's the previous resolved PRs there.

If this isn't making sense, ping me on Slack 😁

**Comment by ewels** [comment]
ok I should have looked around before posting. Just found [this closed PR](https://github.com/nf-core/circrna/pull/48) which still had the option to un-delete the automated sync branch. I restored the `nf-core-template-merge-2.4` branch and that should basically be identical to `TEMPLATE`. 

In fact I'll just rename it. https://github.com/nf-core/circrna/tree/TEMPLATE 👈🏻 there all done 😅

**Review by apeltzer — APPROVED** [review]
Looks good to me and like all the mentioned points are now adressed. Checking quickly if the schema is missing the mimetype thing that Harshil was adding to much pipelines.

File: nextflow_schema.json [hunk 1]
```diff
@@ -9,374 +9,419 @@
...
-                "input",
-                "input_type"
-            ],
+            "description": "Define where the pipeline should find input data and write results.",
+            "required": ["input", "input_type", "outdir"],
             "properties": {
                 "input": {
                     "type": "string",
-                    "fa_icon": "fas fa-dna",
+                    "fa_icon": "fas fa-copy",
```
Comment (apeltzer) [inline nextflow_schema.json:17]: Yeah would be great to have mimetype added here as mentioned in my first review: See an example in viralrecon. 

https://github.com/nf-core/viralrecon/blob/920466a3e4acf540e92950e6d221c20a000a8662/nextflow_schema.json#L18

**Comment by apeltzer** [comment]
@nf-core/core maybe ? 😉😬

**Pull request closed without merging**